troposphere/static/js/components/projects/InstanceDetailsPage.jsx

```diff
@@ -36,8 +39,9 @@ export default React.createClass({
     },
 
     render() {
-        let project = stores.ProjectStore.get(Number(this.getParams().projectId));
-        let instance = stores.InstanceStore.get(Number(this.getParams().instanceId));
+        let projectId = this.context.projectId;
+        let project = stores.ProjectStore.get(projectId);
+        let instance = stores.InstanceStore.get(projectId);
```

Comment:
I have provided a fix for this in: 

https://github.com/cdosborn/troposphere/pull/1